### ISSUE TITLE
What is this really about?

### ISSUE THREAD

**Issue by hoijui** [jpeddicord/speedofsound#45]
Given the README and the GitHub repo description, I figure it has to do what sound, but that's all get.
If you think it makes sense for people to contribute, maybe update the description(s).

**Comment by Lee-Carre** [comment]
@hoijui

> Given the README and the GitHub repo description, I figure it has to do w[ith] sound, but that's all [I] get. If you think it makes sense for people to contribute, maybe update the description(s).

I agree, it's quite ambiguous. The description in F-Droid is somewhat clearer.
Seems that the app was developed not for its own sake as a project, but as a requirement of a comp-sci course (which happened to publish the source on GH). So, the original audience was some university professor, instead of real users, which shows. Hence now being largely abandoned (‘maintenance mode’) by the original author(s).

However, to explain (having used it); the app varies (media) loudness based on GNSS-based speed.

The idea being that, in a vehicle, travelling faster means a noisier environment, and audio playback needs to be louder to overcome this (to clearly hear your favourite music, or navigation voice prompts, audio-book, lecture, or whatever you're listening to). Yet, when travelling slower (or stopped) that same audio level is then too loud.

This app automates the transition between slow & fast, smoothly, based on configurable parameters.
